drivers/serial/Kconfig

```diff
@@ -747,4 +747,43 @@ config UART_BTH4_NPOLLWAITERS
 
 endif # UART_BTH4
 
+menuconfig UART_BTH5
+	bool "BT H5 uart pseudo device"
```

Comment:
```suggestion
	bool "BT H5 UART Pseudo Device"

Reply:
@chengkai15 please fix the remain comment.